handlers/slog/handler.go

```diff
@@ -0,0 +1,50 @@
+package sloghandler
+
+import (
+	"io"
+	"log/slog"
+	"os"
+
+	"github.com/vpakhuchyi/censor"
+)
+
+type config struct {
+	out         io.Writer
+	censor      *censor.Processor
+	ReplaceAttr func(groups []string, a slog.Attr) slog.Attr
+	slog.HandlerOptions
+}
+
+// NewJSONHandler returns a new Slog JSONHandler along with a censor processor. Options can be provided to configure
+// the censor processor and the Slog Handler. If no options are provided, a default configuration is used.
+// See the Option documentation for more details.
+func NewJSONHandler(opts ...Option) *slog.JSONHandler {
+	var cfg config
+	for _, o := range opts {
+		o(&cfg)
+	}
+
+	if cfg.censor == nil {
+		cfg.censor = censor.New()
+	}
+
+	if cfg.out == nil {
+		cfg.out = os.Stdout
+	}
+
+	cfg.HandlerOptions.ReplaceAttr = func(groups []string, a slog.Attr) slog.Attr {
+		attr := a
+		if cfg.ReplaceAttr != nil {
+			attr = cfg.ReplaceAttr(groups, a)
+		}
+
+		switch attr.Key {
+		case slog.TimeKey, slog.LevelKey, slog.SourceKey:
```

Comment:
I would add a comment with an explanation of why Censor execution is not required for these keys.

Reply:
Fixed, thanks